### PR TITLE
review-geoadmin-feature-types

### PR DESCRIPTION
**Pull request by nicholascar**
This vocabulary is auto-generated from the GeoAdmin Features ont:

* http://linked.data.gov.au/def/geoadminfeatures

It's the counterpart to the GeoFeature Types voc which is also auto-generated.

### CODE REVIEW

File: vocabularies/geoadmin-feature-types.ttl [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    skos:broader [ ],
+        <http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> ;
+    skos:definition "A Sub-Block is an administrative sub-division of a Block. Each Block is divided into 25 Sub-Blocks. A Sub-Block is defined as one minute of latitude by one minute of longitude. The Sub-Blocks are prefixed with their corresponding Block name and are labelled from A to Z, excluding I. Labels are allocated from left to right, top to bottom, starting from the northwest corner."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geoadminfeatures> ;
+    skos:prefLabel "Sub-Block"@en .
+
+<http://linked.data.gov.au/def/geofeatures> skos:definition "This vocabulary lists types of geological features relevant to the duties of the Geological Survey of Queensland. It is automatically derived from GSQ's Geologic Features Ontology (http://linked.data.gov.au/def/geofeatures)"@en ;
+    skos:prefLabel "Geologic Feature Types"@en .
+
+<https://orcid.org/0000-0002-8742-7730> a owl:NamedIndividual,
```
Comment (DavidCrosswellGSQ) [inline vocabularies/geoadmin-feature-types.ttl:34]: Please remove this section - you are working for GSQ

File: vocabularies/geoadmin-feature-types.ttl [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+This ontology is part of the Geological Survey of Queensland's set of *component* ontologies that together provide definitions for all of GSQ's data. Of particular importance for this ontology is GSQ's Geological Features Ontology which defines multiple classes of geological feature."""@en ;
+    dcterms:modified "2020-04-15"^^xsd:date ;
+    dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
+    dcterms:title "GSQ Geological Administrative Features Ontology"@en ;
+    owl:imports <http://www.opengis.net/ont/geosparql> ;
+    skos:hasTopConcept <http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> ;
+    sdo:codeRepository <https://github.com/geological-survey-of-queensland/gsq-geoadmin-features-ont> .
+
+<http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> a skos:Concept ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/geoadmin-feature-types.ttl:64]: I don't understand what this concept is doing here.

File: vocabularies/geoadmin-feature-types.ttl [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            sdo:url <https://surroundaustralia.com> ] ;
+    sdo:email <mailto:nicholas.car@surroundaustralia.com> ;
+    sdo:name "Nicholas J. Car" .
+
+<http://linked.data.gov.au/org/gsq> a sdo:Organization ;
+    sdo:email <mailto:geological_info@dnrme.qld.gov.au> ;
+    sdo:name "Geological Survey of Queensland" ;
+    sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> .
+
+<http://linked.data.gov.au/def/geoadminfeatures> a owl:Ontology,
```
Comment (DavidCrosswellGSQ) [inline vocabularies/geoadmin-feature-types.ttl:47]: There is no prefLabel for the conceptScheme

File: vocabularies/geoadmin-feature-types.ttl [hunk 4]
```diff
@@ -0,0 +1,69 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix geof: <http://linked.data.gov.au/def/geofeatures/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix prof: <http://www.w3.org/ns/dx/prof/> .
```
Comment (DavidCrosswellGSQ) [inline vocabularies/geoadmin-feature-types.ttl:4]: The prof: prefix is not used in this vocab

File: vocabularies/geoadmin-feature-types.ttl [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    skos:prefLabel "Block"@en .
+
+<http://linked.data.gov.au/def/geoadminfeatures#Permit> a skos:Concept ;
+    skos:broader <http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> ;
+    skos:definition "Permits are a license granted over a spatial region within the State of Queensland. Permits govern the activities which can be conducted upon them. Activities include exploration, development, and extraction of a specific commodity or commodities."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geoadminfeatures> ;
+    skos:prefLabel "Permit"@en .
+
+<http://linked.data.gov.au/def/geoadminfeatures#SubBlock> a skos:Concept ;
+    skos:broader [ ],
```
Comment (DavidCrosswellGSQ) [inline vocabularies/geoadmin-feature-types.ttl:25]: Is _skos:broader [ ]_ an error?

File: vocabularies/geoadmin-feature-types.ttl [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        <http://linked.data.gov.au/def/geoadminfeatures#AdministrativeFeature> ;
+    skos:definition "A Sub-Block is an administrative sub-division of a Block. Each Block is divided into 25 Sub-Blocks. A Sub-Block is defined as one minute of latitude by one minute of longitude. The Sub-Blocks are prefixed with their corresponding Block name and are labelled from A to Z, excluding I. Labels are allocated from left to right, top to bottom, starting from the northwest corner."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/geoadminfeatures> ;
+    skos:prefLabel "Sub-Block"@en .
+
+<http://linked.data.gov.au/def/geofeatures> skos:definition "This vocabulary lists types of geological features relevant to the duties of the Geological Survey of Queensland. It is automatically derived from GSQ's Geologic Features Ontology (http://linked.data.gov.au/def/geofeatures)"@en ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/geoadmin-feature-types.ttl:31]: Should this be with the conceptSheme?

File: vocabularies/geoadmin-feature-types.ttl (hunk 1 above)
Comment (nicholascar) [reply vocabularies/geoadmin-feature-types.ttl]: I've removed me

File: vocabularies/geoadmin-feature-types.ttl (hunk 2 above)
Comment (nicholascar) [reply vocabularies/geoadmin-feature-types.ttl]: The order's not at all important in this graph: it's just been serialised after the ConceptSscheme due to internal graph ordering.

File: vocabularies/geoadmin-feature-types.ttl (hunk 3 above)
Comment (nicholascar) [reply vocabularies/geoadmin-feature-types.ttl:35]: Added: it was missing due to a fault in the auto-generation script (not updated properly from the GeoFeatures script)

File: vocabularies/geoadmin-feature-types.ttl (hunk 4 above)
Comment (nicholascar) [reply vocabularies/geoadmin-feature-types.ttl]: As above: legacy of copy 'n paste from previous script. Removed.

File: vocabularies/geoadmin-feature-types.ttl (hunk 5 above)
Comment (nicholascar) [reply vocabularies/geoadmin-feature-types.ttl]: Not really an *error* but not at all helpful! It comes from the Class having a restriction which OWL implements as a form of subclass - in this case that a SubBlock be within one and only one Block - which is then translated to a `skos:broader`. 

I've removed this by ensuring that all broader Blank Nodes are filtered out.

File: vocabularies/geoadmin-feature-types.ttl (hunk 6 above)
Comment (nicholascar) [reply vocabularies/geoadmin-feature-types.ttl]: Yes, an artifact of the up-updated build script. Rectified now.

**Pull request merged**